From 030336c1b6524cb3f4174e5de25bd04ebffb6edb Mon Sep 17 00:00:00 2001
From: Yimin Peng <quic_yiminp@quicinc.com>
Date: Fri, 12 Nov 2021 12:17:48 +0800
Subject: [PATCH] systemd-sleep: change suspend states

Try 3 times of deep suspend and then go to s2idle on failure.

Upstream-Status: Pending

Signed-off-by: Yimin Peng <quic_yiminp@quicinc.com>
---
 src/sleep/sleep.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/sleep/sleep.conf b/src/sleep/sleep.conf
index dc2ed37..eb967c5 100644
--- a/src/sleep/sleep.conf
+++ b/src/sleep/sleep.conf
@@ -17,7 +17,7 @@
 #AllowSuspendThenHibernate=yes
 #AllowHybridSleep=yes
 #SuspendMode=
-#SuspendState=mem standby freeze
+SuspendState=mem mem mem freeze
 #HibernateMode=platform shutdown
 #HibernateState=disk
 #HybridSleepMode=suspend platform shutdown
-- 
2.7.4

